### PR TITLE
WIP : Maintenance window resource

### PR DESCRIPTION
**Pull request by nlamirault**
FIX : #62  

Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

### CODE REVIEW

**Comment by dchancogne** [comment]
Any chance this can be merged soon-ish? Thank you.

**Comment by dchancogne** [comment]
Please, pretty please!

**Comment by dchancogne** [comment]
Looks like there has been activity. What remains to be done for this to be merged? Anything I can do that would be useful?

File: pingdom/resource_pingdom_maintenance_window.go [hunk 1]
```diff
@@ -0,0 +1,242 @@
+package pingdom
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
```
Comment (russellcardullo) [inline pingdom/resource_pingdom_maintenance_window.go:8]: Can you change this to use github.com/hashicorp/terraform-plugin-sdk/helper/schema?

File: pingdom/resource_pingdom_maintenance_window.go [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+	windowParams := commonMaintenanceWindowParams{}
+
+	if v, ok := d.GetOk("description"); ok {
+		windowParams.Description = v.(string)
+	}
+	if v, ok := d.GetOk("from"); ok {
+		windowParams.From = v.(int64)
+	}
+	if v, ok := d.GetOk("to"); ok {
+		windowParams.To = v.(int64)
```
Comment (russellcardullo) [inline pingdom/resource_pingdom_maintenance_window.go:87]: I get a panic on this line: `panic: interface conversion: interface {} is int, not int64`.  I think this and the `from` param should probably be changed to `int` as the terraform SDK only has `TypeInt`.

**Review by russellcardullo — CHANGES_REQUESTED** [review]
I get some errors when trying to use this.  Would you mind fixing those as well as updating the README with some documentation on the new resource?

**Comment by nlamirault** [comment]
My bad @russellcardullo @dchancogne I didn't have time to work on it at all. Prefer #98

**Pull request closed without merging**